backend/services/scheduler.go

```
@@ -131,5 +135,68 @@ func TriggerJob(gh utils.GithubClientProvider, ciBackend ci_backends.CiBackend,
+				batch, err = models.DB.GetDiggerBatch(&job.Batch.ID)
+				if err != nil {
+					log.Printf("DiggerJobId %v: Error getting batch: %v", job.DiggerJobID, err)
+					continue
+				}
+				res, err := batch.MapToJsonStruct()
+				if err != nil {
+					log.Printf("DiggerJobId %v: Error getting batch details: %v", job.DiggerJobID, err)
+					continue
+				}
+				// TODO: make this abstract and extracting the right "prService" based on VCS
+				client, _, err := utils.GetGithubService(gh, batch.GithubInstallationId, spec.VCS.RepoFullname, spec.VCS.RepoOwner, spec.VCS.RepoName)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)